### PR TITLE
doc: fix outdated info and broken links

### PR DESCRIPTION
**Pull request by eguzki**

### CODE REVIEW

File: doc/overview.md [hunk 1]
```diff
@@ -7,11 +7,11 @@ Here you’ll learn more about deployment options, environments provided, and ho
 ## Deployment options
 
 You can use APIcast hosted or self-managed, in both cases, it needs connection to the rest of the 3scale API management platform:
-- **<a href="https://support.3scale.net/docs/deployment-options/apicast-cloud-gateway">APIcast hosted</a>**: 3scale hosts the gateway in the cloud. The API gateway is already deployed for you and it's limited to 50,000 calls per day.
-- **APIcast self-managed**: You can deploy APIcast wherever you want. To do so, download the json configuration file from API > Integration > Production or fetch it using the Account management API. The self-managed mode is the intended mode of operation for production environments. Here are a few recommended options to deploy your API gateway:
-  - 'Local' deploy: Install dependencies (check out the [Tools and dependencies info](https://github.com/3scale/apicast#tools-and-dependencies) and get the 'v2.0.0-rc1' tagged version of APIcast (or latest release published).
+- **Embedded APIcast**: Two APIcast gateways (staging and production) come by default with the 3scale API Management installation. They come pre-configured and ready to use out-of-the-box.
```
Comment (dfennessy) [inline doc/overview.md:10]:
```suggestion
- **Embedded APIcast**: A 3scale API Management installation includes two default APIcast gateways, staging and production. These gateways come preconfigured and ready for immediate use.

```

Comment (eguzki) [reply doc/overview.md]: Copied from https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.13/html/installing_3scale/installing-apicast#apicast-deployment-options

It should be updated there as well ;)

File: doc/overview.md [hunk 2]
```diff
@@ -7,11 +7,11 @@ Here you’ll learn more about deployment options, environments provided, and ho
...
-- **<a href="https://support.3scale.net/docs/deployment-options/apicast-cloud-gateway">APIcast hosted</a>**: 3scale hosts the gateway in the cloud. The API gateway is already deployed for you and it's limited to 50,000 calls per day.
-- **APIcast self-managed**: You can deploy APIcast wherever you want. To do so, download the json configuration file from API > Integration > Production or fetch it using the Account management API. The self-managed mode is the intended mode of operation for production environments. Here are a few recommended options to deploy your API gateway:
-  - 'Local' deploy: Install dependencies (check out the [Tools and dependencies info](https://github.com/3scale/apicast#tools-and-dependencies) and get the 'v2.0.0-rc1' tagged version of APIcast (or latest release published).
+- **Embedded APIcast**: Two APIcast gateways (staging and production) come by default with the 3scale API Management installation. They come pre-configured and ready to use out-of-the-box.
+- **Self-managed APIcast**: You can deploy APIcast wherever you want. Here are a few recommended options to deploy APIcast:
```
Comment (dfennessy) [inline doc/overview.md:12]:
```suggestion
- **Self-managed APIcast**: You can deploy APIcast where you prefer. Here are some recommended deployment options:
```

File: doc/overview.md [hunk 3]
```diff
@@ -7,11 +7,11 @@ Here you’ll learn more about deployment options, environments provided, and ho
...
+- **Embedded APIcast**: Two APIcast gateways (staging and production) come by default with the 3scale API Management installation. They come pre-configured and ready to use out-of-the-box.
+- **Self-managed APIcast**: You can deploy APIcast wherever you want. Here are a few recommended options to deploy APIcast:
+  - 'Local' deploy: Install dependencies (check out the [Tools and dependencies info](https://github.com/3scale/apicast#tools-and-dependencies) and get the any version of APIcast (or latest release published).
   - Docker: To avoid having to install APIcast dependencies, you can [download a ready to use dockerized image](https://github.com/3scale/apicast#docker) form our repository.
-  - <a href="openshift-guide.md">OpenShift</a>: APIcast v2.0.0-rc1 runs on top of OpenShift 3.3.
+  - Running APIcast on Red Hat OpenShift: Run APIcast on a supported version of OpenShift. You can connect self-managed APIcasts to a 3scale On-premises installation or to a 3scale Hosted (SaaS) account.
```
Comment (dfennessy) [inline doc/overview.md:15]:
```suggestion
  - Running APIcast on Red Hat OpenShift: Run APIcast on a supported version of OpenShift. You can connect self-managed APIcasts to a 3scale On-Premises installation or to a 3scale Hosted (SaaS) account.

```

**Review by dfennessy — COMMENTED** [review]
Some minor suggestions.

File: doc/overview.md [hunk 4]
```diff
@@ -7,11 +7,11 @@ Here you’ll learn more about deployment options, environments provided, and ho
...
-- **APIcast self-managed**: You can deploy APIcast wherever you want. To do so, download the json configuration file from API > Integration > Production or fetch it using the Account management API. The self-managed mode is the intended mode of operation for production environments. Here are a few recommended options to deploy your API gateway:
-  - 'Local' deploy: Install dependencies (check out the [Tools and dependencies info](https://github.com/3scale/apicast#tools-and-dependencies) and get the 'v2.0.0-rc1' tagged version of APIcast (or latest release published).
+- **Embedded APIcast**: Two APIcast gateways (staging and production) come by default with the 3scale API Management installation. They come pre-configured and ready to use out-of-the-box.
+- **Self-managed APIcast**: You can deploy APIcast wherever you want. Here are a few recommended options to deploy APIcast:
+  - 'Local' deploy: Install dependencies (check out the [Tools and dependencies info](https://github.com/3scale/apicast#tools-and-dependencies) and get the any version of APIcast (or latest release published).
```
[the remark this answers is not in the document]
Comment (eguzki) [reply doc/overview.md]: applied your changes with a minor change in the link

[the remark this answers is not in the document]
Comment (dfennessy) [reply doc/overview.md]: LGTM! :smile:

**Review by dfennessy — APPROVED** [review]

**Pull request merged**